From 5804827db7040f6e26926cd04ac2960521ef489f Mon Sep 17 00:00:00 2001
From: onlykey <onlykey@users.noreply.github.com>
Date: Fri, 15 Nov 2019 15:43:18 -0500
Subject: [PATCH 11/25] Add Onlykey to list

Found a reference to these rules while adding OnlyKey to another set of rules here - https://github.com/snapcore/snapd/blob/master/interfaces/builtin/u2f_devices.go

Signed-off-by: Gustavo B. Schenkel <gustavo.schenkel@gmail.com>
---
 70-u2f.rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/70-u2f.rules b/70-u2f.rules
index 40506d8..1479b9a 100644
--- a/70-u2f.rules
+++ b/70-u2f.rules
@@ -77,4 +77,7 @@ KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="4c4d", ATTRS{idProduct
 # eWBM FIDO2 - Goldengate 310, 320, 500, 450
 KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="311f", ATTRS{idProduct}=="4a1a|4c2a|5c2f|f47c", TAG+="uaccess", GROUP="plugdev", MODE="0660"
 
+# OnlyKey (FIDO2 / U2F)
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="60fc", TAG+="uaccess", GROUP="plugdev", MODE="0660"
+
 LABEL="u2f_end"
-- 
2.32.0

